src/Core/src/Properties/AssemblyInfo.cs

```diff
@@ -10,3 +10,4 @@
 [assembly: InternalsVisibleTo("Microsoft.Maui.UnitTests")]
 [assembly: InternalsVisibleTo("Microsoft.Maui.Core.DeviceTests")]
 [assembly: InternalsVisibleTo("Microsoft.Maui.Controls.Core.UnitTests")]
+[assembly: InternalsVisibleTo("Reloadify-emit")]
```

Comment:
@Clancey tell us more here :)

Reply:
Right now, Reloadify creates a new dll with that namespace. On dotnet core this is not needed. But if you reference something internal (which we still have a lot of internal apis) Then you can't hot reload without this on Mono.  This will go away when we have real HotReload support.